styles/utilities.scss

```
@@ -0,0 +1,23 @@
+.secondary-button-gradient {
+  position: relative;
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border-radius: 16px;
+    padding: 2px;
+    background: linear-gradient(115.82deg, #671bc9 5.15%, #fd0f9e 108.88%);
+    -webkit-mask: linear-gradient(115.82deg, #671bc9 5.15%, #fd0f9e 108.88%) content-box, linear-gradient(#fff 0 0);
+    mask-composite: exclude;
+  }
+  &:hover::before {
+    background: linear-gradient(122.48deg, #671bc9 -52.99%, #fd0f9e 97.26%) !important;
+  }
+  &:focus::before {
+    background: linear-gradient(122.48deg, #671bc9 -52.99%, #fd0f9e 97.26%) !important;
+  }
```

Consider merging the same rules:

```suggestion
  &:focus::before,
  &:hover::before {
    background: linear-gradient(122.48deg, #671bc9 -52.99%, #fd0f9e 97.26%) !important;
  }
```